core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2307,6 +2432,8 @@ object SparkContext extends Logging {
    *
    * @note This function cannot be used to create multiple SparkContext instances
    * even if multiple contexts are allowed.
+   * @param config Spark's config that will be used for initialisation of the `SparkContext`
```

Comment:
`SparkConf` instead of `Spark's config` ? just because elsewhere you're referring to concepts by class name in this line.